### PR TITLE
Update busybox image link to gcr.io for kube-proxy

### PR DESCRIPTION
**Pull request by MrHohn**
**What this PR does / why we need it**: Point busybox image to gcr.io.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #NONE

**Special notes for your reviewer**:
@ixdy @bowei 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/addons/kube-proxy/kube-proxy-ds.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ spec:
         beta.kubernetes.io/kube-proxy-ds-ready: "true"
       initContainers:
       - name: touch-lock
-        image: busybox
+        image: gcr.io/google_containers/busybox:1.24
```
Comment (ixdy) [inline cluster/addons/kube-proxy/kube-proxy-ds.yaml]: please use a newer busybox

Comment (MrHohn) [reply cluster/addons/kube-proxy/kube-proxy-ds.yaml]: Ack, will push a newer image and update here.

**Comment by cblecker** [comment]
/hold
for image update per https://github.com/kubernetes/kubernetes/pull/53819#discussion_r144384731

File: cluster/addons/kube-proxy/kube-proxy-ds.yaml (hunk 1 above)
Comment (MrHohn) [reply cluster/addons/kube-proxy/kube-proxy-ds.yaml]: Updated to `gcr.io/google_containers/busybox:1.27`, corresponding to `busybox:1.27`.

**Comment by cblecker** [comment]
/hold cancel
:)

**Pull request merged**